javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,12 @@
+/* exported filterOutNulls */
+function filterOutNulls(values) {
+  const filteredArray = [];
+  for (let i = 0; i < values.length; i++) {
+    if (values[i] === null) {
+      // console.log();
+    } else {
+      filteredArray.push(values[i]);
+    }
```

Comment:
This would be a good use case for the logical not operator. Since you don't want anything to happen if it is `null`, we only want to check if it's not `null`.

```
if (values[i] !== null) {
  filteredArray.push(values[i])
}
```